index.js

```diff
@@ -109,13 +119,12 @@ CacheBuster.prototype.references = function references() {
         }
 
         var contents = file.contents.toString(encoding);
-
-        var mappings = cachebuster.getRelativeMappings(file.path);
+        var mappings = cachebuster.getRelativeMappings(file.dirname);
         for (var i=0; i < mappings.length; i++) {
             var original = mappings[i].original;
             var cachebusted = mappings[i].cachebusted;
 
-            contents = contents.replace(new RegExp('\\b' + original + '(?!\\.)\\b', 'g'), cachebusted);
+            contents = contents.replace(new RegExp('(["\'])' + original + '(?!\\.)\\b', 'g'), "$1" + cachebusted);
```

Comment:
I needed to change the regex because now original can include `../*` so`\b` is not an option. I need some feedback on this. As far as I can see, all links must be surrounded by quotes, is there any scenario when this is not true?

In case it might help: https://regex101.com/r/GR0E5G/1